rust/toolchain.bzl

```diff
@@ -6,6 +6,27 @@ load(":utils.bzl", "relative_path")
 
 ZIP_PATH = "/usr/bin/zip"
 
+def _get_rustc_env(ctx):
+  version = ctx.attr.version if hasattr(ctx.attr, "version") else "0.0.0"
+  v1, v2, v3 = version.split(".")
+  if "-" in v3:
+    v3, pre = v3.split("-")
+  else:
+    pre = ""
+  return [
+    "CARGO=bazel",  # Bazel is actually performing the build, not cargo
+    "CARGO_MANIFEST_DIR=\"$(basename \"$PWD/%s\")\"" % ctx.build_file_path,
```

Comment:
I'm lukewarm on including this. This arg has weird semantics, and anybody trying to use it probably is going to have a bad/weird time in a bazel context. Consider that a BUILD file may not be in the same place that the corresponding Cargo.toml would be.

Are you agreeable to omitting this for now and addressing the lack of this arg when the problem presents itself?

Reply:
Sounds good, I just need the version for now.